src/datepicker/__tests__/datepicker-i18n-chinese.scenario.js

```diff
@@ -0,0 +1,21 @@
+/*
+Copyright (c) 2018-2020 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+
+import * as React from 'react';
+
+import {StatefulCalendar} from '../index.js';
+import {zhCN} from 'date-fns/locale/index.js';
```

Comment:
FYI it doesn't look like the snapshot tester supports unicode (characters are displaying as black boxes).